gridcomps/Cap/FlapCLI.F90

```diff
@@ -7,42 +7,30 @@ module MAPL_FlapCLIMod
    use FLAP
    use mapl_KeywordEnforcerMod
    use mapl_ExceptionHandling
-   use mapl_CapOptionsMod, only:  MAPL_CapOptions !Rename is for backward compatibility. Remove renaming for 3.0
+   use mapl_CapOptionsMod, only:  MAPL_CapOptions
    implicit none
    private
 
    public :: MAPL_FlapCLI
-   public :: MAPL_CapOptions !Needed for backward compatibility. Remove for 3.0
 
-   type :: MAPL_FlapCLI
+   type :: MAPL_FlapCLI_Type
```

Comment:
```suggestion
   type :: FlapCLI_Type
```
Now that we've resolved the main issue, we may as well fix stylistic things for MAPL 3.   I'd really like to even avoid the `_Type` suffix, but let's not waste too much time debating it.